### PR TITLE
Replace mobile logo with Ably + docs logo

### PR DESCRIPTION
**Pull request by aralovelace**
> _NOTE TO REVIEWERS_ - please **do not review PRs in the `DRAFT` state**, as the PR may change substantially before it is ready to review. Thanks.

## Description

A PR description indicating the purpose of the PR.

* [EDX-482](https://ably.atlassian.net/browse/EDX-482?atlOrigin=eyJpIjoiNzU0OWI3MGY4NDg1NGJjMmFkNjdhMDVmNWYwNWQ4N2EiLCJwIjoiaiJ9)
* [EDX-495](https://ably.atlassian.net/browse/EDX-495?atlOrigin=eyJpIjoiY2MzMTNlNzkwZmQzNGM2MmE3YjVkNTFkOWQ5MGU5MGEiLCJwIjoiaiJ9)
* Also added feedback from Lenka 
* Fixed spacing issue Code Snippet Horizontal Menu
* Fix Logo Link

## Review

Instructions on how to review the PR. 

* check the page for tablet and mobile

### CODE REVIEW

File: src/components/Header/TopMainNav/TopMainNavIllustration/TopMainNavAblyLogo.tsx [hunk 1]
```diff
@@ -13,7 +13,7 @@ export const TopMainNavAblyLogo = ({ href = '/' }: { href?: string }) => {
       {width > 1024 ? (
         <StaticImage src="/images/icons/ably-docs-logo.svg" />
       ) : (
-        <StaticImage className="h-32" src="/images/icons/ably-logo.svg" />
+        <StaticImage className="h-32" src="/images/icons/ably-docs-logo-mobile.svg" />
```
Comment (MymmiJ) [inline src/components/Header/TopMainNav/TopMainNavIllustration/TopMainNavAblyLogo.tsx:16]: Slight scope creep, but can you make sure both these images don't resize by giving them a fixed width and height as well?

Comment (aralovelace) [reply src/components/Header/TopMainNav/TopMainNavIllustration/TopMainNavAblyLogo.tsx]: @MymmiJ  updated and got approval from Lenka for the position of the logo

**Review by snikidev — APPROVED** [review]
